python/torch_mlir/csrc/reference_lazy_backend/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ mlir_configure_python_dev_packages()
 # Library definition
 ###########################################################################
 
-set(LIBRARY_OUTPUT_PATH  "${TORCH_MLIR_PYTHON_PACKAGES_DIR}/torch_mlir/torch_mlir/reference_lazy_backend")
+set(LIBRARY_OUTPUT_PATH  "${TORCH_MLIR_PYTHON_PACKAGES_DIR}/torch_mlir/torch_mlir/_mlir_libs")
```

Comment:
Maybe add a comment here "We piggyback on the shared library setup/infra used by Torch-MLIR Python bindings for consistency."